### PR TITLE
Working version of NameLayer with Mercury

### PR DESCRIPTION
**Pull request by suirad**
Also added tracking of all online players for other plugins to utilize(civchat). I would appreciate someone testing the previously added mercury synced commands.
After a review, this should be pull/Civtest ready.

### CODE REVIEW

**Comment by suirad** [comment]
@rourke750 @ttk2

**Comment by suirad** [comment]
I'm working on Civchat now

File: src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java [hunk 1]
```diff
@@ -49,5 +88,29 @@ else if (reason.equals("transfer")){
...
+		MercuryPlugin.handler.sendMessage("all", "name_layer", "logoff "+event.getPlayer().getDisplayName());
+		Bukkit.getLogger().info("LOGOFF");
+	}
 
+	@Override
+	public void run() {
+		NameLayerPlugin nl = NameLayerPlugin.getInstance();
+		nl.isMercuryEnabled = Bukkit.getPluginManager().isPluginEnabled("Mercury");
+		if (nl.isMercuryEnabled){
+			nl.getServer().getPluginManager().registerEvents(this, nl);
```
Comment (rourke750) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: add this here: getServer().getPluginManager().registerEvents(new MercuryMessageListener(), this);

Also I'll end up refactoring this so we dont need the delay but its okay for now.  I need to read up more on the documentation.

Comment (suirad) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: You want it to make another MercuryMessageListener? How come? Wouldn't nl.getServer().getPluginManager().registerEvents(this, nl); increment the reference so it wouldnt get gc'd when the runnable is over?

Comment (rourke750) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: I copied and pasted my bad. Just register the class.

Comment (suirad) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: Is that not what line 109 is doing?

Comment (rourke750) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: Ignore me lol.

**Comment by rourke750** [comment]
ok to test

File: src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java (hunk 1 above)
Comment (suirad) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: Rourke go back to sleep lol

Comment (rourke750) [inline src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java]: I was actually being rushed out the house and trying to get you sorted lol.
Guh NameLayer master has so many changes to live.

**Comment by ttk2** [comment]
so do we want to move this to testing? Also roruke do you want to try and sync dev and live versions for a bit while we move to the latest spigot build?

**Comment by rourke750** [comment]
ok to test

**Comment by ttk2** [comment]
do you want this on Civtest?

**Comment by rourke750** [comment]
No point right now since no shard support.  Also have other stuff I need to change in this before it is ready.

**Pull request merged**